app/helpers/brand_logo_helper.rb

```diff
@@ -0,0 +1,16 @@
+module BrandLogoHelper
+  def render_brand_logo
+    return unless should_render_logo?
+    render 'application/brand_logo'
+  end
+
+  private
+
+  def should_render_logo?
+    current_organization&.id == branded_organization_id
```

Comment:
Due to a bug in how we use the `current_organization_id` session key we need to make sure that we show the logo only in pages where the user is logged in.

```suggestion
    return false unless current_user
    current_organization&.id == branded_organization_id
```